shared/deployments/check-publish.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+git diff --quiet HEAD~ HEAD -- $PWD/contracts
+
+if [ $? == "1" ]
+then
+  aragon apm publish major --files dist/ --environment continuous-deployment --apm.ipfs.rpc https://ipfs.autark.xyz:5001 --ipfs-check false --propagate-content false --skip-confirmation --no-prepublish
+  exit $?
+else 
+  echo "no contract changes"
+fi
+
+git diff --quiet HEAD~ HEAD -- $PWD/app $PWD/../../shared/identity $PWD/../../shared/lib $PWD/../../shared/store-utils $PWD/../../shared/ui $PWD/../../shared/utils
```

Comment:
It seems a little bit strange that every single app's `publish:cd` process will check this same set of directories (other than `PWD/app`). But maybe that's a good thing?

Reply:
Yeah this makes it much easier to maintain this flow. The other directories are all shared, and out of an abundance of caution, and due to the fact that minor and patch publishes cost very little gas and are quick, it makes sense to publish all apps in CD if a shared file is updated.